weblate_web/templates/snippets/service.html

```diff
@@ -1,107 +1,167 @@
 {% load i18n %}
+{% load static %}
+{% load date_diff %}
 
-            <div class="payment-form">
-                <div class="form-line">
-                    <div class="line-left">{% trans "Service" %}</div>
+<div class="payment-form pt-pb">
+    <div class="form-line style6 bdr-btm">
+        <div class="service-variant">
+            <p>{{ service.get_status_display }}</p>
+        <div class="service-time">
+            {% with subscriptions=service.subscription_set.all %}
+            {% for subscription in subscriptions %}
+            {% if subscription.expires|days_diff_from_today < 30 %}
+            {% if subscription.expires|days_diff_from_today < 2 and subscription.expires|days_diff_from_today >= 0 %}
+            <t class="orange">{{ subscription.expires|days_diff_from_today}} day remaining</t>
+            {% elif subscription.expires|days_diff_from_today < 0  %}
+            <t class="orange">Expired</t>
+            {% else %}
+            <t class="orange">{{ subscription.expires|days_diff_from_today}} days remaining</t>
+            {% endif %}
+            {% else %}
+            <t class="grey">{{ subscription.expires|days_diff_from_today}} days remaining</t>
+            {% endif %}
+            {% endfor %}
+            {% endwith %}
+        </div>
+        </div>
+    </div>
+                <div class="form-line clearr" style="height:20px;background: linear-gradient(180deg, #E9EAEC -45%, #FFFFFF 100%);border:none;"></div>
+
+                <div class="form-line style6">
+                    <div class="line-left">{% trans "Users" %}</div>
                     <div class="line-right">
-                        {{ service.get_status_display }}
-                        {% if not service.check_in_limits %}
-                        <div class="alert error">
-                        {% trans "Exceeds selected service offering." %}
-                        </div>
-                        {% endif %}
+                    {% for owner in service.users.all %}
+                    <form id="server_user_{{owner.id}}_form" method="post" action="{% url 'service-user' pk=service.pk %}">
+                    <div class="user-slot">
+                        <div class="user-slot-content">
+                        <p>{{ owner.last_name }}</p>
+                    <t>{{ owner.email }}</t></div>
+                    {% if owner == user %}
+                    <div class="user-img"><img src="{% static "img/Shield.png" %}"></div>
+                    {% else %}
+                    {% csrf_token %}
+                    <img onclick="document.getElementById('server_user_{{owner.id}}_form').submit()" class="user-img" name="remove" value="remove" src="{% static "img/delete.png" %}">
```

Comment:
Please avoid inline JS, it will not work on production due to CSP.

Reply:
I have moved the inline JS to custom.js file.